pandas/tests/arrays/test_datetimes.py

```diff
@@ -90,3 +90,10 @@ def test_setitem_clears_freq(self):
                                         tz='US/Central'))
         a[0] = pd.Timestamp("2000", tz="US/Central")
         assert a.freq is None
+
```

Comment:
is this near other repeat tests

Reply:
There are no other repeat tests AFAICT; I think those are in the tests.extension part of 24024

Reply:
```
(pandas) bash-3.2$ grep -r test_repeat pandas/tests
pandas/tests/series/test_analytics.py:    def test_repeat(self):
pandas/tests/extension/base/methods.py:    def test_repeat(self, data, repeats, as_series, use_numpy):
pandas/tests/extension/base/methods.py:    def test_repeat_raises(self, data, repeats, kwargs, error, msg, use_numpy):
Binary file pandas/tests/extension/base/__pycache__/methods.cpython-36.pyc matches
pandas/tests/io/test_stata.py:    def test_repeated_column_labels(self):
pandas/tests/test_multilevel.py:    def test_repeat(self):
pandas/tests/test_strings.py:    def test_repeat(self):
pandas/tests/indexes/multi/test_reshape.py:def test_repeat():
pandas/tests/indexes/period/test_period.py:    def test_repeat_freqstr(self, index, use_numpy):
pandas/tests/indexes/common.py:    def test_repeat(self):
pandas/tests/indexes/datetimes/test_ops.py:    def test_repeat_range(self, tz_naive_fixture):
pandas/tests/indexes/datetimes/test_ops.py:    def test_repeat(self, tz_naive_fixture):
pandas/tests/indexes/timedeltas/test_ops.py:    def test_repeat(self):
pandas/tests/indexes/test_base.py:    def test_repeat(self):
```

Reply:
right, i meant repeat tests for DTA/TDA/PA.  There are more coming up in the tests.extension part of 24024.

Reply:
ok, i guess that 's fine. need a followup then to consolidate DTI/DTA test for common methods at some point then.